run.sh

```diff
@@ -143,6 +143,21 @@ elif [ $SYSTEMTESTS_SCENARIO = "REMOTE_CONFIG_MOCKED_BACKEND_ASM_DD" ]; then
     export SYSTEMTESTS_LOG_FOLDER=logs_remote_config_mocked_backend
     export SYSTEMTESTS_LIBRARY_PROXY_STATE='{"mock_remote_config_backend": "ASM_DD"}'
 
+elif [ $SYSTEMTESTS_SCENARIO = "REMOTE_CONFIG_MOCKED_BACKEND_FEATURES_NOCACHE" ]; then
+    export RUNNER_ARGS="scenarios/remote_config/test_remote_configuration.py::Test_RemoteConfigurationFields scenarios/remote_config/test_remote_configuration.py::Test_RemoteConfigurationUpdateSequenceFeaturesNoCache"
+    export SYSTEMTESTS_LOG_FOLDER=logs_remote_config_mocked_backend
```

Comment:
can you give a different name for log folder for each scenario (we'll later collect logs artifact)